nameable.rb

```diff
@@ -0,0 +1,11 @@
+module Nameable
+    attr_accessor :Nameable
+
+    def initializing (nameable)
```

Comment:
- [ ] After making Nameable a class, I am still not able to run your code, I get this error wrong number of arguments (given 2, expected 1) (ArgumentError). I suggest that instead of def initializing, you make it def initialize and remove the space before the parentheses.